README.md

```diff
@@ -2,6 +2,85 @@
 
 Bot for Scratch Addons Discord server
 
-[Cloud hosted on opeNode.io](https://www.openode.io/)
+Cloud hosted on [opeNode.io](https://www.openode.io/)
+
+## Features
+### General
+---
+### PotatoBoard
+Messages with more than 6 potato emoji reactions are shown in the #potatoboard channel.
+
+* #### Related: /explorepotatos - Get a random message from the potatoboard.
+
+### React to messages including certain strings 
+Scradd reacts to messages that include any of the following strings with an emoji associated with the found string:
+
+* dango, dangos, dangoes
+* potato, potatos, potatoes
+* griff
+* amongus
+* sus
+* appel, appels, appeles
+* tera, teras, teraes
+* give you up, give up
```

Comment:
noo keep these secret 😭 

you can mention scradd does this, but don't say what messages

yes they can look in the code but keep it more cryptic lol